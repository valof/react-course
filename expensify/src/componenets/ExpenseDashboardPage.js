import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilter from './ExpenseListFilter';

const ExpenseDashboardPage = () => (
    <div>
        <ExpenseListFilter x = "10" />
        <ExpenseList />
    </div>
);

export default ExpenseDashboardPage;
