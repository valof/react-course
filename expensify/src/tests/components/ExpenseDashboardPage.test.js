import React from 'react'
import ExpenseDashboardPage from '../../componenets/ExpenseDashboardPage';
import ShallowRenderer from 'react-test-renderer/shallow';

test ('should render Expense Dashboard page correctly', () => {
    const testRenderer = ShallowRenderer.createRenderer()
    const result = testRenderer.render(
            <ExpenseDashboardPage />
      );
      expect(result).toMatchSnapshot();
})
