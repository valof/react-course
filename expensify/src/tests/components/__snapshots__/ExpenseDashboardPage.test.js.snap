// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Dashboard page correctly 1`] = `
<div>
  <Memo(Connect(ExpenseFilterList)) />
  <Memo(Connect(ExpenseList)) />
</div>
`;
